contributors/devel/sig-architecture/feature-gates.md

```diff
@@ -152,17 +179,70 @@ other action outside of Kubernetes to use it. This gives some grace period for
 users to take action, but such feature gates will eventually set
 `LockToDefault` to `true` and then be retired, like normal.
 
-[After at least two releases post-GA and deprecation](https://kubernetes.io/docs/reference/using-api/deprecation-policy/#deprecation),
-feature gates should be removed. Typically, we add a comment in
+[After at least three releases where a feature has been locked to the default value, feature gates should be removedpost-GA/deprecation and locked to
```

Comment:
"removedpost-GA" missing a space and a linewrap?

Edit: actually I think you forgot to delete something?  This reads as:

"""
After at least three releases ... feature gates should be removed post-GA ...,  feature gates should be removed.
"""

Maybe just:  `After three releases where a feature has been locked to the default value (whether that feature is GA or deprecated), feature gates should be removed.` ?